Core/App.Web/Settings/AuthSettings.cs

```diff
@@ -10,5 +12,8 @@ public class AuthSettings
         public string AuthenticationType { get; set; } = "oauth2";
         public string ApiKeyHeader { get; set; } = "X-Api-Key";
         public string ApiKey { get; set; }
+        public string[] AllowedIssuers { get; set; } = Array.Empty<string>();
```

Comment:
@jstorteboom should I create a new AzureAuthSettings class that inherits this one or is it okay to add these three new props in the existing class?

Reply:
I'd like to be explicit, so an AzureAuthSettings would be nicer. Also, AuthSettings doesn't mean anything, should be OAuthSettings. AzureAuthSettings can of course derive from OAuthSettings